omas/omas_physics.py

```diff
@@ -267,9 +267,15 @@ def summary_total_powers(ods, update=True):
               'ec': copy.deepcopy(q_init), 'lh': copy.deepcopy(q_init),
               'ic': copy.deepcopy(q_init)}
 
+    ignore_indices = list(range(100, 108)) + list(range(900,910))
+
     for time_index in sources[0]['profiles_1d']:
         vol = sources[0]['profiles_1d'][time_index]['grid']['volume']
         for source in sources:
+            if sources[source]['identifier.index'] in ignore_indices:
+                # Skip the combined sources to prevent double counting
+                print(sources[source]['identifier.name'], "is excluded from summary_total_powers with index", sources[source]['identifier.index'])
```

Comment:
If you know that's the right thing to do, I'd be ok not to `print` anything on screen.

Reply:
@orso82  Right now nobody is using the combined sources, 101 - 109 but I am worried that in the future these might be populated instead of the individual ones (Ic & ec combined), this this also excludes self-made indices 901-909. 

But I agree it would be annoying if it is being printed to the screen so I will delete it 👍